src/libraries/System.Net.Http/src/System/Net/Http/Headers/HeaderDescriptor.cs

```diff
@@ -263,11 +264,13 @@ private static bool TryDecodeUtf8(ReadOnlySpan<byte> input, [NotNullWhen(true)]
             decoded = null;
             return false;
         }
+    }
 
-        // QPack Static Table
+    internal static class QPackStaticTable
```

Comment:
Makes sense.  Can you move this to its own file?

Reply:
Done